.circleci/config.yml

```diff
@@ -30,6 +30,47 @@ jobs:
           paths:
             - .venv
             - integreat_compass.egg-info
+  npm-install:
+    docker:
+      - image: "cimg/node:lts"
+    resource_class: small
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - npm-{{ checksum "package-lock.json" }}-v2
+      - run:
+          name: Install npm dependencies
+          command: |
+            if [[ -d "node_modules" ]]; then
+              echo "Node modules restored from cache, skipping npm install"
+            else
+              npm ci
+              if [[ -n $(git diff --shortstat package-lock.json) ]]; then
+                echo "package-lock.json is out of date:"
+                git --no-pager diff package-lock.json
+                exit 1
+              fi
```

Comment:
```suggestion
```
This does not make sense - `npm ci` does not rewrite the the lock file, so it will never be out of date (and on the other hand: if we use `npm install` here, it would not indicate a problem, since the generated lock file depends on the system it's generated on and is not a single source of truth)...
Probably, we should change this in the integreat-cms as well...